sign-pkg-windows/sign.py

```diff
@@ -60,8 +57,8 @@ def sign(executable, *, vault_uri, cert_name, client_id, client_secret, tenant_i
                '-kvt', tenant_id,
                '-kvs', client_secret,
                '-kvc', cert_name,
-               '-tr',  'http://timestamp.digicert.com',
-               '-v',   executable]
+               '-tr', 'http://timestamp.digicert.com',
+               '-v', executable]
```

Comment:
It's really not clear to me that responding to flake8 "errors" improves code quality.

Reply:
They generally do, but in cases like this an exception to the rule ([E241](https://www.flake8rules.com/rules/E241.html)) could be made on a line, file or project level. One can also disable/enable linting complete for a block of code with `#noqa`.